test/test_readonly_task.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require "syskit/test/self"
+    
+module Syskit
+    task_service "ReadOnlyTask" do
+        def initialize
+            super
+            @read_only = true
+        end
+    end
+
+    describe ReadOnlyTask do
+        it "skips configuration" do
+            task_m = TaskContext.new_submodel do
+                property "p", "/double"
+            end
+
+            task = syskit_stub_and_deploy(task_m.with_arguments(read_only: true))
+
+            assert_raises(RuntimeError) do
+                task.properties.p = 2.0
+            end
+
+            syskit_configure(task)
+
+            expect_execution { task.start! }
+                .to { emit task.start_event }
+        end
```

Comment:
I know it's a draft, but I think you could copy/paste the four tests I wrote on the chat, just to make sure you won't forget one ...

Reply:
done